service/core.go

```diff
@@ -99,6 +99,7 @@ type serviceCore struct {
 	metricsCore
 	tracerCore
 	authClient     auth.Client
+	metricsClient  metrics.Client
```

Comment:
Why do we need this on service core? I don't see us using metrics client being used anywhere except for initialization.